newrelic.yml

```diff
@@ -413,6 +413,14 @@ common: &default_settings
 # May be one of [auto|prepend|chain|disabled].
 # instrumentation.grpc_server: auto
 
+# Specifies a list of hostname patterns separated by commas
+# for gRPC hosts to not trace requests for
+# By default, New Relic's own infinite tracing servers are
+# listed with this pattern: 'tracing\.(?:staging-)?edge\.nr-data'
+# It is recommended that that pattern be kept and added to if
+# additional host patterns need to be defined.
+# instrumentation.grpc.host_denylist: ""
```

Comment:
@kaylareopelle idea: Let's put the 'nr-data' pattern into a constant that can not be altered. Then we permit the configuration to specify additional ignore patterns. By default, there are no additional patterns. When present, the additional patterns will be added to the list that always contains at least the 'nr-data' pattern. This would simplify the config and documentation and prevent any unwanted unblocking of infinite tracing monitoring.

Thoughts?

Reply:
I like that a lot. If we like how this pans out, there may be other settings that could benefit from similar measures.

Reply:
Addressed with [5f30f9e](https://github.com/newrelic/newrelic-ruby-agent/pull/1283/commits/5f30f9e0794726d7b2bad11287f7874c678a2fa5)